package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "automagically-action-execute",
-  "version": "2.0.0",
+  "version": "2.0.2",
```

Comment:
I would honestly just go with `0.0.0` or something here, we will just keep forgetting it and we won't release the npm "package" ever.